src/main/java/org/folio/service/file/definition/FileDefinitionServiceImpl.java

```diff
@@ -84,4 +85,10 @@
       : FileDefinition.UploadFormat.CSV;
   }
 
+  private String getFileNameByRequest(QuickExportRequest request) {
+    return StringUtils.isEmpty(request.getFileName())
+      ? QUICK_EXPORT + CSV_FILE_FORMAT
+      : request.getFileName() + CSV_FILE_FORMAT;
```

Comment:
what if the file name given is test.csv?  we'll be appending csv twice?

Reply:
or infact because this is the filename of download what if user gives test.mrc?

Reply:
we expect fileName, not fileName with extension. If the user will provide such a name, for example, test.mrc, the result will be 
![image](https://user-images.githubusercontent.com/60778653/102240612-be49f580-3f00-11eb-9914-7069ccd1c7dd.png)